### PR TITLE
Remove deprecated method Content::fillParserOutput()

### PR DESCRIPTION
**Pull request by samwilson**
Move the HTML-setting into the ContentHandler instead of the Content.

Bug: #698

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
What are the compat implications of this change?

**Comment by samwilson** [comment]
> What are the compat implications of this change?

It bumps the required MW version to 1.38. How important is it to maintain backwards compatibility? I think if it's necessary, then probably a separate legacy class of Content will be needed, because I don't think it's possible to optionally override a method (and so avoid the deprecation).

**Comment by JeroenDeDauw** [comment]
Then this will be for the next major release of Maps: version 11. Will hold off with merging into master for a little bit.

**Comment by JeroenDeDauw** [comment]
You could remove the MW 35-MW37 test runs form GitHub actions in this PR

Would also help to have the tests run and pass against MW 39, though that might require fixing things out of scope of this PR, and thus better done seperately

**Comment by samwilson** [comment]
I'm attempting to keep 1.35 compatibility. Feels like it's not too complicated... see what you think!

File: src/GeoJsonPages/GeoJsonContentHandler.php [hunk 1]
```diff
@@ -4,18 +4,49 @@
...
 
 	protected function getContentClass(): string {
-		return GeoJsonContent::class;
+		return version_compare( MW_VERSION, '1.38', '<' ) ? GeoJsonLegacyContent::class : GeoJsonContent::class;
 	}
 
 	public function makeEmptyContent(): GeoJsonContent {
-		return new GeoJsonContent( GeoJsonContent::newEmptyContentString() );
+		$class = $this->getContentClass();
+		return new $class( $class::newEmptyContentString() );
```
Comment (JeroenDeDauw) [inline src/GeoJsonPages/GeoJsonContentHandler.php:25]: Why is this change needed?

Comment (samwilson) [reply src/GeoJsonPages/GeoJsonContentHandler.php:25]: It's to avoid having to repeat the version check here, to make sure it returns an object of the right class (either GeoJsonContent or GeoJsonLegacyContent). This way, it won't have to be changed when eventually the legacy content class is removed.
